packages/ui/src/components/StoryPublicationDate/StoryPublicationDate.tsx

```diff
@@ -1,26 +1,28 @@
-import type { Culture, Story } from '@prezly/sdk';
-import { type AlgoliaStory, getStoryPublicationDate } from '@prezly/theme-kit-core';
+import type { Culture } from '@prezly/sdk';
 import { format } from 'date-fns';
 import * as locales from 'date-fns/locale';
```

Comment:
Loading all locales on client side will probably blow the bundle size. 
Let's leave a `FIXME` comment to look into this later.